src/olympia/stats/management/commands/download_counts_from_file.py

```diff
@@ -25,7 +25,7 @@ def is_valid_source(src, fulls, prefixes):
     a prefix in the list of valid prefix sources.
 
     """
-    return src in fulls or any(p in src for p in prefixes)
+    return src in fulls or any(src.startswith(p) for p in prefixes)
```

Comment:
This is an actual change because, as I understood it, the existing code was incorrect.

Reply:
@diox I'm a bit wary of _any_ changes in our stats pipeline... so does should I be "fixing" this?

Reply:
dropped in https://github.com/mozilla/addons-server/pull/14293/commits/cf07efd5fd0363daaf3e326cf687bb37009bca9d